gremlin-test/src/main/resources/org/apache/tinkerpop/gremlin/test/features/sideEffect/Inject.feature

```diff
@@ -217,4 +217,28 @@ Feature: Step - inject()
       | d[100].i |
       | d[300].i |
       | d[1].i |
-      | d[3].i |
\ No newline at end of file
+      | d[3].i |
+
+  @GraphComputerVerificationInjectionNotSupported
+  Scenario: g_injectX1_3_100_300X_list
+    Given the modern graph
+    And the traversal of
+      """
+      g.inject([1, 3, 100, 300])
+      """
+    When iterated to list
+    Then the result should be unordered
+      | result |
+      | l[d[1].i,d[3].i,d[100].i,d[300].i] |
+
+  @GraphComputerVerificationInjectionNotSupported
+  Scenario: g_injectX1_3_100_300X_set
+    Given the modern graph
+    And the traversal of
+      """
+      g.inject({1, 3, 100, 300})
```

Comment:
Do we have defined semantics for duplicate elements passed in the set initializer? My expectation is that they just get dropped. If so could we add a test for
```
g.inject({1, 1}) => {1}
```